terraform/modules/kubernetes/monitoring.tf

```diff
@@ -115,3 +115,48 @@ resource "azurerm_monitor_metric_alert" "redis_memory" {
     ]
   }
 }
+data "azurerm_monitor_diagnostic_categories" "default" {
+  count = var.deploy_azure_backing_services ? 1 : 0
+
+  resource_id = azurerm_postgresql_flexible_server.postgres-server[0].id
+}
+
+resource "azurerm_log_analytics_workspace" "pg_loganalytics" {
+  count = var.deploy_azure_backing_services ? 1 : 0
+  name                = "${azurerm_postgresql_flexible_server.postgres-server[0].name}-pgsql-logs"
```

Comment:
According to [microsoft](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations#management-and-governance) the suffix should be "-log"